src/fundus/publishers/de/motorsport_magazin.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class MotorSportMagazinParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector: CSSSelector = CSSSelector("section.article-body")
```

Comment:
This selector is a bit too coarse, the whole text is extracted as a whole blob. 
```suggestion
        _paragraph_selector: CSSSelector = CSSSelector("section.article-body > p")
```